src/modules/Accordion.js

```diff
@@ -30,7 +30,11 @@ export default class Accordion extends Component {
     let subHeadlineMarkup;
     if (subHeadline) {
       subHeadlineMarkup = (
-        <Paragraph size="large" margin="none" align="center">
+        <Paragraph
+          className={`${CLASS_ROOT}__subHeadline`}
```

Comment:
From what I have seen, Grommet core uses hyphens to separate words (kebab-case) instead of camelCase for CSS class names.